### ISSUE TITLE
$.nmManual + auto resize / reposition 

### ISSUE THREAD

**Issue by tommykamkcm** [nyroDev/nyroModal#192]
Hi All

I have a question about the difference between $('.nyroModal').nyroModal(); and $.nmManual(url);.

Basically, I would like to keep the nyroModal popup always in the middle of the screen.
I tried to resize the browser and observe the behaviour. Results below:

$('.nyroModal').nyroModal(): Reposition will automatically be triggered on browser resize.
$.nmManual(url): Reposition will never be triggered.

So, my question is: Is it possible to enable the auto-reposition on browser when using $.nmManual(url)?

**Comment by bpmn** [comment]
I have the same question.....

**Comment by daverupp** [comment]
Any news for this question? Don't get it to reposition the nyroModal.

**Comment by Blackskyliner** [comment]
same problem when clicking on a link inside nyromodal, it will load but brick the auto resize and reposition.

found a somewhat dirty solution:

```
$(window).smartresize(function(){
                        $.nmTop().getInternal()._resize();
                        $.nmTop().resize(true);
                    });
```

also posted @ #202

**Comment by nyroDev** [comment]
This has been fixed with commit 0d162edcdfd3d4e47fb7e778f2347ba62b505c97
